.gitignore

```diff
@@ -61,3 +61,7 @@ bash.exe.stackdump
 package-lock.json
 
 !/dist/package.json
+/.vs/ngx-datetimepicker/v17
+/.vs/slnx.sqlite
+/.vs/VSWorkspaceState.json
+/.vs/ngx-datetimepicker/config
```

Comment:
Just curious, where did these gitignore files come from?

Also, are you opening this project in Visual Studio? Its fine if that's what you want to use, but I do strongly encourage you to try out vscode.